bootstrap/main.tf

```diff
@@ -2,46 +2,38 @@ locals {
   argocd_chart = yamldecode(file("${path.module}/../charts/argocd/Chart.yaml")).dependencies.0
 }
 
-resource "helm_release" "argocd" {
-  name       = "argocd"
-  repository = local.argocd_chart.repository
-  chart      = local.argocd_chart.name
-  version    = local.argocd_chart.version
-
-  namespace         = "argocd"
-  dependency_update = true
-  create_namespace  = true
-  timeout           = 10800
-  values            = [for i in concat([local.helm_values.0.argo-cd], var.helm_values) : yamlencode(i)]
-}
-
-resource "time_static" "iat" {}
-
-resource "random_uuid" "jti" {}
-
+# argocd secret key for token generation, it should be passed to next argocd generation
 resource "random_password" "argocd_server_secretkey" {
   length  = 32
   special = false
 }
 
+# jwt token for `pipeline` account
 resource "jwt_hashed_token" "argocd" {
   algorithm   = "HS256"
-  secret      = local.argocd_server_secretkey
+  secret      = random_password.argocd_server_secretkey.result
   claims_json = jsonencode(local.jwt_token_payload)
 }
 
-resource "random_password" "oauth2_cookie_secret" {
-  length  = 16
-  special = false
-}
+resource "time_static" "iat" {}
 
-resource "random_password" "argocd_server_admin" {
-  length  = 16
-  special = false
-}
+resource "random_uuid" "jti" {}
+
+resource "helm_release" "argocd" {
+  name       = "argocd"
+  repository = local.argocd_chart.repository
+  chart      = local.argocd_chart.name
+  version    = local.argocd_chart.version
+
+  namespace         = "argocd"
```

Comment:
```suggestion
  namespace         = var.namespace
```

Reply:
`namespace` variable is missing in variables.tf

Reply:
can argocd bootstrap be installed in another namespace? does this even make sense? 
same way that can loki-stack be installed in something different from namespace "loki-stack" ?

Reply:
fixed